examples/seismic/self_adjoint/sa_02_iso_implementation2.ipynb

```diff
@@ -974,11 +974,145 @@
    "execution_count": 13,
    "metadata": {},
    "outputs": [
+    {
```

Comment:
You probably didn't really mean to execute this cell (and, in general, to introduce any change to this notebook) ?

Reply:
I am not sure how to figure out which cell? Do I load the file in an editor to fine line 977? Can you add cell# in reviews easily?

Reply:
oh no sorry I simply meant to ask why the edit in this notebook? I wouldn't have expected a change here